modules/test/playwright/pages/style-book-web/StyleBooksPage.ts

```diff
@@ -40,22 +54,49 @@ export class StyleBooksPage {
 	}
 
 	async deleteStyleBook(styleBookName: string) {
-		await this.page
-			.locator(
-				'input[name=_com_liferay_style_book_web_internal_portlet_StyleBookPortlet_keywords][type=search]'
-			)
-			.fill(styleBookName);
+		await this.searchStyleBook(styleBookName);
+
+		await this.page.getByLabel('More actions').click();
+		await this.page.getByRole('menuitem', {name: 'Delete'}).click();
+
+		await this.page.getByRole('button', {name: 'Delete'}).click();
+	}
 
-		await this.page.getByTitle('Search for', {exact: true}).click();
+	async editStyleBook(styleBookName: string) {
+		await this.searchStyleBook(styleBookName);
+
+		await this.page.getByLabel('More actions').click();
+		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
+	}
+
+	
+	async searchStyleBook(styleBookName: string) {
```

Comment:
@ethib137  I disagree calling this `searchFor`. That gives the idea that you will pass in the type of element/entity/thing you will search.

We could call this `search` or `find`.

Reply:
Agree with @thiago-buarqque suggestion on [this comment](https://github.com/liferay-platform-experience/liferay-portal/pull/724#discussion_r1768690014).

Reply:
Hey @ilzamcmed and @thiago-buarqque remember to always check existing usages to see if there's a pattern already. In this case we currently have two instances of this called `searchFor` and two instances of this called `search`. Which one we go with doesn't matter too much to me, but we need to keep it consistent. So, since you want to go for `search` we need to change [our usage in feature flags](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/feature-flag-web/pages/FeatureFlagsInstanceSettingsPage.ts#L40) too. Additionally, you should either make the change in [documentLibraryPage](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts#L218) and ask the team to review it on this PR, or send them a separate PR to ensure consistency.

Reply:
Let's keep it as `searchFor` then. We don't need to change in all of those places just because we are adding in here.

Reply:
Well actually, in either case we are now aware of this inconsistency. So we should be making sure it's consistent no matter what. If we go with `searchFor` we still have two instances of `search` that should be changed. In that case two of those instances are owned by other teams so it would require more review by other teams then going with `search`. I'd recommend going with `search` and making the changes I recommended.

Reply:
Thanks @ethib137 !
I'll change the feature flag here and send another PR for document library page

Reply:
PR created! https://github.com/liferay-platform-experience/liferay-portal/pull/732